src/chatdlg.cpp

```diff
@@ -67,6 +67,10 @@ CChatDlg::CChatDlg ( QWidget* parent ) :
         SLOT ( OnClearChatHistory() ), QKeySequence ( Qt::CTRL + Qt::Key_E ) );
 
     pMenu->addMenu ( pEditMenu );
+#if defined ( Q_OS_IOS )
+    QAction *action = pMenu->addAction (tr ( "&Close" ));
+    connect(action, SIGNAL(triggered()), this, SLOT(close()));
```

Comment:
Code style: Jamulus is a bit strange concerning this, but you'll need to add spaces after "connect" and after ( . See the other methods.